src/components/ContactBox.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
This is largely copied and pasted over from WholePageBox. Consider making WholePageBox component take in an optional prop boxType that is an enum:
```TypeScript
enum BoxType {
    FULL_WIDTH,
    HALF_WIDTH
}
```
If boxType is HALF_WIDTH, apply a class called halfBoxBody containing styles for the box.